tools/base-language/lib/Metrics.ts

```diff
@@ -18,32 +18,44 @@
 import { Metric, MetricType, SeriesType } from "@syntest/metric";
 
 export enum SeriesName {
-  PATHS_COVERED = "paths-covered",
   BRANCHES_COVERED = "branches-covered",
-  EXCEPTIONS_COVERED = "exceptions-covered",
+  STATEMENTS_COVERED = "statements-covered",
   FUNCTIONS_COVERED = "functions-covered",
-  LINES_COVERED = "lines-covered",
```

Comment:
Lines and paths can also be calculated for the actual coverage numbers right?

Reply:
Yes, but the current instrumentation does not support this so I left them out.